models/bamf_nnunet_mr_liver/utils/BamfProcessorRunner.py

```diff
@@ -0,0 +1,84 @@
+"""
+-------------------------------------------------
+MedicalHub - Run Module for ensembling nnUNet inference.
+-------------------------------------------------
+-------------------------------------------------
+Author: Rahul Soni
+Email:  rahul.soni@bamfhealth.com
+-------------------------------------------------
+"""
+
+from mhubio.core import Instance, InstanceData, DataType, FileType, CT, SEG
+from mhubio.core import Module, IO
+import os, numpy as np
+import SimpleITK as sitk
+from skimage import measure, filters
+import numpy as np
+import shutil
+
+
+@IO.ConfigInput('in_data', 'nifti:mod=ct|mr', the="input data to run BamfProcessor on")
```

Comment:
See the comments I added regarding BamfProcessor in [PR50](https://github.com/MHubAI/models/pull/50).